### PR TITLE
Added Docker PHP

### PR DESCRIPTION
**Pull request by Nyholm**
Ping @joelwurtz, Do you want it here?

### CODE REVIEW

**Comment by joelwurtz** [comment]
Fine with that yes

File: index.html [hunk 1]
```diff
@@ -154,6 +154,16 @@
 		<div class="row">
             <div class="col-md-4"></div>
             <div class="col-md-1">
+                    <img src="https://avatars2.githubusercontent.com/u/16955543?v=3&s=50" alt="Docker PHP" />
+            </div>
+			<div class="col-md-7">
+				<b><a href="https://github.com/docker-php/docker-php">Docker PHP</a></b>
+				<br /> A Docker client written in PHP.
```
Comment (sagikazarmark) [inline index.html:161]: You are aware that `<br>` is valid HTML5, right? :wink:

**Comment by sagikazarmark** [comment]
Thanks

**Comment by sagikazarmark** [comment]
Uh, image is a little bit large.

File: index.html (hunk 1 above)
Comment (Nyholm) [reply index.html:161]: Is it? I was not aware of that.

**Comment by Nyholm** [comment]
I'll add some css to prevent this.

**Pull request merged**